R/visr_plot.R

```diff
@@ -92,10 +92,10 @@ visr.survfit <- function(
  ,y_ticks = NULL
  ,fun = "surv"
  ,legend_position = "right"
+ ,theme = "visR"
```

Comment:
Our current thinking was to set a global theme instead of using a theme argument for every plot. We also agreed no more features as of last week.